easybuild/easyconfigs/g/GangSTR/GangSTR-2.4.4-GCCcore-10.3.0.eb

```diff
@@ -0,0 +1,32 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Pau Ruiz <pau.ruiz@hpcnow.com>, Alex Garcia
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+name = 'GangSTR'
+version = '2.4.4'
+
+homepage = 'https://github.com/gymreklab/GangSTR'
+description = """GangSTR is a tool for genome-wide profiling tandem repeats from short reads"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```

Comment:
As this depends on `GSL-2.7-GCC-10.3.0.eb` and `HTSlib-1.12-GCC-10.3.0.eb` this should also use the `GCC` toolchain (as `GCC` level items can use `GCCcore` items as dependencies but not the reverse). So:

1. `toolchain = {'name': 'GCCcore', 'version': '10.3.0'}`
2. change the filename to be `GangSTR-2.4.4-GCCcore-10.3.0.eb`